### PR TITLE
add .annotations to configure annotations of Deployment's level

### PR DESCRIPTION
**Pull request by begemotik**
Addresses https://github.com/centrifugal/helm-charts/issues/83

### CODE REVIEW

**Comment by begemotik** [comment]
@FZambia would appreciate your time reviewing it.

File: charts/centrifugo/values.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ nameOverride: ""
 fullnameOverride: ""
 namespaceOverride: ""
 priorityClassName: ""
+annotations: {}
```
Comment (FZambia) [inline charts/centrifugo/values.yaml:24]: Let's add comment above: `# Annotations to be added to the deployment`

File: charts/centrifugo/templates/deployment.yaml [hunk 2]
```diff
@@ -6,6 +6,12 @@ metadata:
   namespace: {{ include "centrifugo.namespace" . }}
   labels:
     {{- include "centrifugo.labels" . | nindent 4 }}
+  {{- if .Values.annotations }}
```
Comment (FZambia) [inline charts/centrifugo/templates/deployment.yaml:9]: Let's use `with` - like [here](https://github.com/centrifugal/helm-charts/blob/f47f7d7ea45e5a88e1343e9ebfd7a54dec91d4ee/charts/centrifugo/templates/service.yaml#L11)

**Review by FZambia — COMMENTED** [review]
Thank you! Added couple of comments

**Review by FZambia — APPROVED** [review]

**Pull request merged**